PocketKit/Sources/PocketKit/MyList/ItemsList/ItemsListViewController.swift

```diff
@@ -4,7 +4,8 @@ import CoreData
 import Combine
 import Kingfisher
 
-class ItemsListViewController<ViewModel: ItemsListViewModel>: UIViewController, UICollectionViewDelegate, UICollectionViewDataSourcePrefetching {
+class ItemsListViewController<ViewModel: ItemsListViewModel>: UIViewController, UICollectionViewDelegate, UICollectionViewDataSourcePrefetching, UISearchBarDelegate {
```

Comment:
do we still need to make this change if we move it to the SavesContainerVC?

Reply:
youre right. good catch